server/lib/time/get-current-time.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Returns the current time, in milliseconds. This function mainly exists so it can be injected into
+ * classes that needs to get the current time and also be easily testable.
+ */
+export function getCurrentTime(): number {
```

Comment:
I think in general, unless you *have* to inject a function for some reason, creating a class is just a lot cleaner and doesn't require manual registration or `@inject(name)` things